bitbucket/bitbucket.go

```diff
@@ -82,9 +82,11 @@ func (hook Webhook) ParsePayload(w http.ResponseWriter, r *http.Request) {
 
 	if len(hook.uuid) > 0 {
 		if uuid != hook.uuid {
```

Comment:
This should be done outside the function in a generic way, instead of here (I think)